Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -260,23 +260,32 @@ In order to address these attack vectors, the outcome of the following checks sh
 
 #### Overview
 
--- TODO [Provide a general description of the issue "Testing WebView Protocol Handlers".] --
+Protocol handler is a basic form of IPC in iOS system. Basically, it allows for invoking arbitrary applications, by calling a URL scheme with specified parameters. There are some default handlers<sup>[1]</sup>, however a developer is allowed to register his own custom URL scheme<sup>[2]</sup>. Unfortunately, this brings a serious security concern, what found out developers of Skype application<sup>[3]</sup>. The Skype application registered `skype://` protocol handler, which allows for making a call to arbitrary number without asking for a user's permission. Attackers exploited this vulnerability by putting an invisible `<iframe src=”skype://xxx?call"></iframe>` (where `xxx` was replaced by a premium number), so any Skype's user who visited malicious website unconsciously was forced to call to a premium number.
```

Comment:
@Sjord thanks for your comments - I'll fix my mistakes.

@sushi2k now I'm a little bit confused with those test cases. I was going to describe 'file://' in a 'Testing for Local File Inclusion in WebViews [iOS]'. Could you please specify what other protocol handlers should be mentioned in this section ('Testing WebView Protocol Handlers')?